moesifwsgi/workers.py

```diff
@@ -173,3 +175,45 @@ def stop(self):
         # Clear workers
         self.batcher = None
         self.workers = []
+
+
+class ConfigJobScheduler:
+
+    def __init__(self, debug, config):
+        self.DEBUG = debug
+        self.scheduler = None
+        self.config = config
+
+    def exit_config_job(self):
+        try:
+            # Shut down the scheduler
+            self.scheduler.remove_job('moesif_config_job')
+            self.scheduler.shutdown()
```

Comment:
add `self.scheduler = None` too?

Reply:
It'd be fine since this would be trigger on job exit. So no need to set it to None